### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by giscus** [WyattJia/WyattJia#81] [Announcements]
# about/

https://eventloop.live/about/

**Comment by cxybreader** [comment]
Good to know you a little but since this website is special for those programming person, we'd like to know what kind of program you had participated and what program you're working on?which area you are specializing in? Last but not least, keep on updating~

**Reply by WyattJia** [reply]
Of course, thx for your attention.

**Comment by 0MUQX4Q** [comment]
🙌
